less/components/expander.less

```diff
@@ -21,6 +21,9 @@ html.no-js {
     font-weight: bold;
     text-transform: uppercase;
     padding: 2rem;
+    margin: 0;
+    font-size: inherit;
```

Comment:
Oh yea what you said makes sense @toolness !  

I guess I'm just wondering if we can remove those 2 `font-size` rules altogether.  

- For the Clubs Toolkit page, it's kinda odd to me that we specifically set a "header" type of text just 1px smaller than default body copy size (16px) and that rule only applies on a single page on our site.  (maybe it was falsely set or things got changed along the way?)
- For the `.expander-header` class, `font-size: inherit` essentially means set the font size to 16px now.

For these reasons I'm thinking if we should just remove those CSS rules. @sabrinang thoughts?